internal/xds/rbac/rbac_engine.go

```diff
@@ -131,9 +139,37 @@ func newEngine(config *v3rbacpb.RBAC) (*engine, error) {
 		}
 		policies[name] = matcher
 	}
+
+	auditOptions := config.GetAuditLoggingOptions()
+	auditLoggers := []*audit.Logger{}
+	if auditOptions != nil {
+		for _, logger := range auditOptions.LoggerConfigs {
+			auditLoggerFactory := audit.GetLoggerBuilder(logger.AuditLogger.Name)
+			if auditLoggerFactory == nil {
+				if logger.IsOptional {
+					continue
+				}
+				return nil, fmt.Errorf("No builder registered for %v", logger.AuditLogger.Name)
+			}
+			customConfig := logger.AuditLogger.TypedConfig
```

Comment:
TypedConfig is protobuf.Any, which contains the typed_url and value. I think it's the value here that we need to pass into ParseLoggerConfig, not the Any itself.